WcaOnRails/config/locales/en.yml

```diff
@@ -522,6 +522,10 @@ en:
       invalid: "Invalid email, WCA ID, or password."
       not_found_in_database: "Invalid email, WCA ID, or password."
       unconfirmed: "You have to confirm your email address before continuing. Didn't receive a confirmation email? Go to https://www.worldcubeassociation.org/users/confirmation/new to request a new one."
+    #context: Overridden key for Devise reconfirmation email subject
+    mailer:
+      confirmation_instructions:
+        subject: "Email reconfirmation instructions"
```

Comment:
Maybe we could have a less robotic sounding phrase here? Maybe something like "please confirm your new email address"?

Reply:
Sure, will do!